app/test_main.py

```diff
@@ -1,3 +1,20 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("input_params, expected_output", [
+    ((0, 0), [0, 0]),
+    ((14, 14), [0, 0]),
+    ((15, 15), [1, 1]),
+    ((23, 23), [1, 1]),
+    ((24, 24), [2, 2]),
+    ((27, 27), [2, 2]),
+    ((28, 28), [3, 2]),
+    ((100, 100), [21, 17]),
```

Comment:
add some test for negative numbers and that if the function receives an incorrect type of data it raises the correct exception.